aws/resource_aws_lambda_function.go

```diff
@@ -820,6 +821,8 @@ func resourceAwsLambdaFunctionUpdate(d *schema.ResourceData, meta interface{}) e
 			if err != nil {
 				return fmt.Errorf("Error setting concurrency for Lambda %s: %s", d.Id(), err)
 			}
+		} else {
+			return fmt.Errorf("Error processing concurrency for Lambda %s. reserved_concurrent_executions should be >=0 for limiting concurrency, reserved_concurrent_executions = 0 for disabling this lambda from being triggered and reserved_concurrent_executions = -1 for removing concurrency constraint. Supplied value: %d", d.Id(), nc.(int))
```

Comment:
This can be removed with `ValidateFunc: validation.IntAtLeast(-1),` on the attribute 👍

Reply:
@bflad Thanks for the review :) Will make the necessary changes.